pkgs/by-name/li/linien-common/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "linien-common";
```

Comment:
can linien-common be used as standalone package ? is there some binaries ? ( sorry I havent tried building this yet )

Reply:
There's a `setup.py` for the `linien-common`. I think it can be a standalone package.

Reply:
buildPythonApplication is for applications and this is a library

Reply:
I guess it would be better to move linien-common to python-modules and leave the other three in pkgs/by-name.